packages/arb-bridge-peripherals/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "arb-bridge-peripherals",
   "version": "1.0.0",
   "license": "Apache-2.0",
+  "private": true,
```

Comment:
I don't believe we want to make this package private. We can instead create the openzeppelin workspace path in the top level package.json which is already private

Reply:
this do was done for hoisting. r.e. workspace path, does yarn let you do that? ("foist" a package down to a particular package's node modules?)